packages/core/adder/options.ts

```diff
@@ -1,40 +1,47 @@
 import { type OptionValues as CliOptionValues, program } from "commander";
 import { AdderDetails, AddersExecutionPlan } from "./execute.js";
-import { booleanPrompt, textPrompt } from "../utils/prompts.js";
+import { booleanPrompt, selectPrompt, textPrompt, type PromptOption } from "../utils/prompts.js";
 
-export type BooleanDefaultValue = {
+export type BooleanQuestion = {
     type: "boolean";
     default: boolean;
 };
 
-export type StringDefaultValue = {
+export type StringQuestion = {
     type: "string";
     default: string;
 };
 
-export type NumberDefaultValue = {
+export type NumberQuestion = {
     type: "number";
     default: number;
 };
 
+export type SelectQuestion<Value = any> = {
+    type: "select";
+    default: Value;
+    options: PromptOption<Value>[];
+};
+
 export type BaseQuestion = {
     question: string;
+    // TODO: we want this to be akin to OptionValues<Args> so that the options can be inferred
```

Comment:
I assume this was overlooked. Or is it something you didn't get working?

Reply:
heh, yea I wasn't able to figure it out that day and never got around to fixing it. i'll take another crack at it in a few

Reply:
i changed it _a tiny bit_ so that it isn't just a flat out `any`, but I still couldn't figure this one out. i'm not sure if there's a way to derive the prop types of an object that the condition is a child of.

Reply:
See #397